socialNetwork/src/UserTimelineService/UserTimelineService.cpp

```diff
@@ -114,7 +120,22 @@ int main(int argc, char *argv[]) {
                            std::make_shared<TBinaryProtocolFactory>());
     LOG(info) << "Starting the user-timeline-service server with Redis Cluster support...";
     server.serve();
-  } else {
+  }
+  else if (init_redis_replica_client_pool) {
+      Redis redis_replica_client_pool = init_redis_replica_client_pool(config_json, "redis_replica");
```

Comment:
keep consistent naming, `redis_replica` and `redis-primary` are not consistent, see other config entries and fix naming to follow same pattern